### ISSUE TITLE
paddle.fluid.layers.kldiv_loss(x, target, reduction='mean', name=None)

### ISSUE THREAD

**Issue by paulpaul91** [PaddlePaddle/Paddle#17632]
paddle 平台上这个接口说明文档上的公式错了，请更正

**Comment by wangguibao** [comment]
请问您看到的页面是哪个，方便给下链接吗，我们检查一下

**Comment by paulpaul91** [comment]
> 请问您看到的页面是哪个，方便给下链接吗，我们检查一下

http://paddlepaddle.org/documentation/docs/zh/1.4/api_cn/layers_cn.html#kldiv-loss

**Comment by wangguibao** [comment]
检查了一下，文档对kldiv loss的描述应该没问题？
请问您说的文档中公式错了，具体是哪里不对呢？我们再确认一遍

**Comment by paulpaul91** [comment]
> 检查了一下，文档对kldiv loss的描述应该没问题？
> 请问您说的文档中公式错了，具体是哪里不对呢？我们再确认一遍

应该是l(x,y)=y∗(log(y)−log(x))

**Comment by heavengate** [comment]
@paulpaul91  您好，kldiv_loss公式应该是没有错的，pytorch也是这样实现的，https://pytorch.org/docs/stable/nn.html?highlight=kldiv#torch.nn.KLDivLoss

**Comment by paulpaul91** [comment]
> @paulpaul91 您好，kldiv_loss公式应该是没有错的，pytorch也是这样实现的，https://pytorch.org/docs/stable/nn.html?highlight=kldiv#torch.nn.KLDivLoss
难道你们就没有怀疑pytorch也是抄错了呢，请看下定义
